UnitImplementations/UIComponents/LogIn/LogIn.ts

```diff
@@ -0,0 +1,22 @@
+import {generateWhiteButton, generateRedButton} from '../../../Src/UIComponents/Buttons/Buttons';
+import {generateTextInput} from '../../../Src/UIComponents/FormComponent/FormComponent'
+import {createElement} from '../../../Src/UIComponents/utils/utils'
+
+const container = createElement('div', 'main-container');
+const title = createElement('p', 'main-title'); 
+title.appendChild(document.createTextNode('Log In'));
+
+container.appendChild(title);
+document.body.appendChild(container);
+
+const input = generateTextInput('Nickname', 'name');
+container.appendChild(input);
+
+const button = generateRedButton('Log In', ()=>{});
+
+button.style.width = "200px";
+button.style.height = "50px";
```

Comment:
I suggest to add to .css file selector .main-container > button{width: 200px; height: 50px}
and remove this code.

Reply:
Thanks, I changed it.